### ISSUE TITLE
Texture name is not generated correctly when osc inputs have more than 3 informations

### ISSUE THREAD

**Issue by almina-orange** [fand/veda#238]
Hello,

I tried to send osc messages from some apps like ZIGSIM, but VEDA has no response and the texture remain black.

So I checked 3 types of osc inputs, like below...
```
/foo (--> osc_foo)
/foo/bar (--> osc_foo_bar)
/foo/bar/hoge (--> osc_foo_bar_hoge)
```

I was able to successfully receive the above 2 patterns, but last one was not.
I don't know where is wrong. How to do when osc inputs have more than 3 informations?

btw thanks for your work, I love it.

**Comment by fand** [comment]
Thanks @almina-orange !

uh, it was a typical regex bug 😭 
just fixed and published VEDA v2.13.2. plz try it out!

**Comment by almina-orange** [comment]
Hi @fand ! It works. thanks!
